gent_base.theme

```diff
@@ -36,6 +36,19 @@ function gent_base_form_alter(&$form, FormStateInterface $form_state, $form_id)
   }
 }
 
+/**
+ * Implements hook_preprocess().
+ */
+function gent_base_preprocess_page(&$variables) {
+  $routeMatch = \Drupal::routeMatch();
+  $entityTypeManager = \Drupal::entityTypeManager();
+  $entity = $routeMatch->getParameter('node');
+
+  $headerImage = $entity->get('field_header_image')->view();
```

Comment:
Je zal hier wel de nodige if's moeten plaatsen om geen error te krijgen als het veld niet gekend is in de entity ;-)

Reply:
Fixed.